integrationtests/as-lucene-directory/src/test/java/org/infinispan/test/integration/as/wildfly/controller/MemberRegistration.java

```diff
@@ -32,18 +38,34 @@ public void register() throws Exception {
    }
 
    public void register(Member member) throws Exception {
-      em.persist(member);
+      Search.getFullTextEntityManager(em).persist(member);
       initNewMember();
```

Comment:
'em.persist(member);' should be enough, you only need to wrap the EntityManager when needing to access methods which are not exposed on the JPA API.